source/exe/signal_action.h

```diff
@@ -79,8 +82,10 @@ class SignalAction : NonCopyable {
    * This constant array defines the signals we will insert handlers for.
    *
    * Essentially this is the list of signals that would cause a core dump.
+   * The object will contain an array of struct sigactions with the same number
+   * of elements that are in this array.
    */
-  static constexpr int FATAL_SIGS[] = {SIGABRT, SIGBUS, SIGFPE, SIGILL, SIGSEGV};
+  static const constexpr int FATAL_SIGS[] = {SIGABRT, SIGBUS, SIGFPE, SIGILL, SIGSEGV};
```

Comment:
My C++ foo isn't strong enough to understand the meaning of `const constexpr`, I'm genuinely curious to learn what's behind this.